lib/elastomer/client/delete_by_query.rb

```diff
@@ -0,0 +1,82 @@
+module Elastomer
+  class Client
+
+    class DeleteByQuery
+
+      def initialize( client, query, params = nil )
```

Comment:
Ok, I will remove those spaces. Shall we make a sweep through the whole repo with a whitespace normalization commit as a separate PR?